--- origsrc/gst-plugins-ugly-0.10.19/ext/amrnb/amrnbdec.h	2011-12-30 11:40:45.000000000 -0600
+++ src/gst-plugins-ugly-0.10.19/ext/amrnb/amrnbdec.h	2012-03-13 22:29:20.052393600 -0500
@@ -22,7 +22,7 @@
 
 #include <gst/gst.h>
 #include <gst/audio/gstaudiodecoder.h>
-#include <interf_dec.h>
+#include <opencore-amrnb/interf_dec.h>
 
 G_BEGIN_DECLS
 
--- origsrc/gst-plugins-ugly-0.10.19/ext/amrnb/amrnbenc.h	2011-12-30 11:40:45.000000000 -0600
+++ src/gst-plugins-ugly-0.10.19/ext/amrnb/amrnbenc.h	2012-03-13 22:28:19.804947700 -0500
@@ -21,7 +21,7 @@
 #define __GST_AMRNBENC_H__
 
 #include <gst/gst.h>
-#include <interf_enc.h>
+#include <opencore-amrnb/interf_enc.h>
 #include <gst/audio/gstaudioencoder.h>
 
 G_BEGIN_DECLS
--- origsrc/gst-plugins-ugly-0.10.19/ext/amrwbdec/amrwbdec.h	2011-12-30 11:40:45.000000000 -0600
+++ src/gst-plugins-ugly-0.10.19/ext/amrwbdec/amrwbdec.h	2012-03-13 22:28:54.885954200 -0500
@@ -22,8 +22,8 @@
 
 #include <gst/gst.h>
 #include <gst/audio/gstaudiodecoder.h>
-#include <dec_if.h>
-#include <if_rom.h>
+#include <opencore-amrwb/dec_if.h>
+#include <opencore-amrwb/if_rom.h>
 
 G_BEGIN_DECLS
 
